app/client/jsx/LibraryRoom.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+
+class LibraryRoom extends Component {
```

Comment:
We've been trying to switch to using functional components. You can rewrite this pretty easily as a function. See `Poke.jsx` for an example

Reply:
Done (I think I did that correctly? It works, anyhow).

Reply:
Looks good!